src/test_workflow/integ_test/integ_test_suite_opensearch.py

```diff
@@ -139,8 +139,18 @@ def custom_node_endpoint_encoder(node_endpoint: NodeEndpoint) -> dict:
             logging.info(f"{script} does not exist. Skipping integ tests for {self.component.name}")
             return 0
 
+    @property
+    def test_report_dirs(self) -> list[str]:
+        return ["integTest", "integrationTest", "integTestRemote"]
+
     @property
     def test_artifact_files(self) -> dict:
-        return {
-            "opensearch-integ-test": os.path.join(self.repo_work_dir, "build", "reports", "tests", "integTest")
-        }
+        default_report_path = os.path.join(self.repo_work_dir, "build", "reports", "tests", "integTest")
```

Comment:
In that case, why not just walk through all possibilities, is there still a need for the default?

Thanks.

Reply:
This should be faster as the default standard is `integTest`, if found `integTest` dont loop for the reports. In the recent commit peter I have removed the `integTest` from `additional_test_report_dirs`, this should be a better approach.
Thanks